README.md

```diff
@@ -155,7 +155,7 @@ ${DOCKER} run \
     -v $(pwd):/dest \
     golang:1.16.4 \
     sh -c "
-    go install github.com/NVIDIA/mig-parted/cmd@latest
+    go install github.com/NVIDIA/mig-parted/cmd/nvidia-mig-parted@latest
     mv /go/bin/cmd /dest/nvidia-mig-parted
```

Comment:
Could it be that the `cmd` here is not correct? Running `go install` will mostlikely create a `nvidia-mig-parted` exectuable in the go path, so the following should be `mv /go/bin/nvidia-mig-parted /dest/nvidia-mig-parted`.